contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -333,11 +360,24 @@ def _transcribe(item, parents, editRate, masterMobs):
                 otio.opentime.RationalTime(media_start, editRate),
                 otio.opentime.RationalTime(media_length, editRate)
             )
-            # copy the metadata from the master into the media_reference
-            media.metadata["AAF"] = mastermob_child.metadata.get("AAF", {})
+
+            # Copy the metadata from the master into the media_reference
+            clipMetadata = mastermob_child.metadata.get("AAF", {})
```

Comment:
Since the code is modifying the dictionary later, it should be deep copied here. That way the modification doesn't accidentally get included if/when this same mastermob_child is used elsewhere.

Reply:
Yes, I can change that. I still think the mastermob_child will not be used anywhere else, but better safe than sorry, it should not be expensive. Just pointing out: I kept the original behavior, just added an intermediate step storing the result in a variable.